addons/mail/tests/discuss/test_discuss_channel_access.py

```diff
@@ -233,16 +233,16 @@ def test_10_discuss_channel_member_access(self):
             ("public", "group_matching", "outside", "other", "write", False),
             ("public", "group_matching", "outside", "other", "unlink", False),
             ("public", "group_failing", "member", "self", "create", False),
-            ("public", "group_failing", "member", "self", "read", False),
+            ("public", "group_failing", "member", "self", "read", True),
```

Comment:
Changes in access is not ok, the added sudo is not ok.

Reply:
Isn't a channel member able to read their own record?

Reply:
Not if group_failing